undertow/src/main/resources/org/wildfly/extension/undertow/LocalDescriptions.properties

```diff
@@ -27,7 +27,8 @@ undertow.configuration=null
 undertow.default-virtual-host=The default virtual host to use for deployments
 undertow.default-servlet-container=The default servlet container to use for deployments
 undertow.default-server=The default server to use for deployments
-undertow.instance-id=The cluster instance id
+undertow.instance-id=The cluster instance id (defaults to {$jboss.node.name} if undefined)
+undertow.obfuscate-instance-id=Obfuscate the instance-id and server name when routing
```

Comment:
@pferraro  now, the question that remains is it a problem to call it obfuscate-instance-id when we are actually obfuscating the whole route? If it is, would 'obfuscate-route' attribute name be easy enough for the final user to understand the purpose of the attribute? Not obfuscating the server-name would defeat the usage of it as a salt.

Reply:
@pferraro  that makes more sense to me. I'll update the PR with this.